left-right/src/read.rs

```diff
@@ -198,7 +198,9 @@ impl<T> ReadHandle<T> {
     /// Note that it is only safe to read through this pointer if you _know_ that the writer will
     /// not start writing into it. This is most likely only the case if you are calling this method
     /// from inside a method that holds `&mut WriteHandle`.
-    pub fn raw_handle(&mut self) -> Option<NonNull<T>> {
+    ///
+    /// _Writing_ through this handle using any kind of interior mutability is never safe.
```

Comment:
I think the way to phrase this is that it is never safe to cast the returned pointer to `&mut`. It is not necessarily unsound to mutate through an inner `UnsafeCell` for example.